lib/storage/client.js

```diff
@@ -1044,6 +1047,65 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * Make a request object for the create group request
+   * @param {string} name
+   * @param {Array} capabilities
+   * @param {KeyPair} encryptionKeyPair
+   *
+   * @return {Group}
+   */
+  async createGroupRequest(name, capabilities = [], encryptionKeyPair) {
+    const groupKeyPair = await this.crypto.generateKeypair()
+    const eak = await this.crypto.encryptPrivateKey(
+      groupKeyPair.privateKey,
+      encryptionKeyPair.privateKey,
+      encryptionKeyPair.publicKey
+    )
+    let groupData = new GroupData(name, capabilities)
+    let membershipKeys = new GroupMembershipKeys(
+      encryptionKeyPair.publicKey,
+      eak
+    )
+    let group = new Group(groupData, null, membershipKeys)
+    return group
+  }
+
+  /**
+   * createGroup creates a group and sends it to TozStore.
+   *
+   * @param {string} name  the name of the group
+   * @param {array} capabilities an array of strings of the capabilities the group will have
+   *
+   * @returns {Group} A response from TozStore; the group that has been written.
+   */
+  async createGroup(name, capabilities = []) {
+    // /* eslint-enable */
+    const encryptionKeyPair = new KeyPair(
+      this.config.publicKey,
+      this.config.privateKey
+    )
+    let createGroupReq = await this.createGroupRequest(
+      name,
+      capabilities,
+      encryptionKeyPair
+    )
+    let response = await this.authenticator.tsv1Fetch(
+      `${this.config.apiUrl}/v2/storage/groups`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: createGroupReq.stringify(),
+      }
+    )
+    let storedGroupResp = await checkStatus(response)
```

Comment:
there is an additional helper which checks status and decodes to JSON. Not all methods use it because it is a somewhat newer addition to utilities, but it will clean up some intermediate vars here.